ern-local-cli/src/index.ts

```diff
@@ -55,21 +57,36 @@ function showInfo() {
 }
 
 function showVersion() {
-  // get electrode-native local-cli version
-  if (config.get('platformVersion')) {
-    log.info(`ern-local-cli : ${config.get('platformVersion')}`)
-  }
-  // get electrode-native global-cli version
-  const packageInfo = JSON.parse(
-    execSync('npm ls -g electrode-native --json').toString()
-  )
-  if (packageInfo && packageInfo.dependencies) {
-    log.info(
-      `electrode-native : ${
-        packageInfo.dependencies['electrode-native'].version
-      }`
+  let packageInfo
+  try {
+    const pkg = path.join(
+      execSync('yarn global dir')
+        .toString()
+        .trim(),
+      'node_modules',
+      'electrode-native',
+      'package.json'
     )
+    packageInfo = fs.existsSync(pkg) ? require(pkg) : ''
+  } catch (err) {
+    log.trace(err)
   }
+
+  if (!packageInfo) {
+    try {
+      packageInfo = JSON.parse(
+        execSync(`npm ls -g electrode-native --json`).toString()
```

Comment:
we do not need  template strings, can work with 'npm ls -g electrode-native --json'

Reply:
D'oh, of course. Will change. Thanks!